heft-plugins/heft-webpack4-plugin/package.json

```diff
@@ -14,34 +14,31 @@
   "scripts": {
     "build": "heft build --clean",
     "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
   },
   "peerDependenciesMeta": {
     "@types/webpack": {
       "optional": true
-    },
-    "@types/webpack-dev-server": {
-      "optional": true
     }
   },
   "peerDependencies": {
     "@rushstack/heft": "^0.45.14",
     "@types/webpack": "^4",
-    "@types/webpack-dev-server": "^3",
     "webpack": "~4.44.2"
   },
   "dependencies": {
     "@rushstack/node-core-library": "workspace:*",
-    "webpack-dev-server": "~3.11.0"
+    "@types/tapable": "1.0.6",
+    "tapable": "1.1.3",
+    "webpack-dev-server": "~4.9.3"
```

Comment:
No longer a peer dependency?

Reply:
Never was; the types were the only peer dependency.